dizoo/beergame/main/beergame_test.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
rename to `entry/beergame_eval.py`, which is the standard format, and you should refer to other envs if you meet the same case.